falkordb-cluster/cluster-entrypoint.sh

```
@@ -75,6 +75,68 @@
+  if [[ -f "$DATA_DIR/nodes.conf" && -s "$DATA_DIR/nodes.conf" ]];then
+    discrepancy=0
+    while IFS= read -r line;do
+      if [[ $line =~ .*@0.* || $line =~ .*fail.* ]];then
+        discrepancy=$(( $discrepancy + 1 ))
+        hostname=$(echo $line | awk '{print $2}' | cut -d',' -f2| cut -d':' -f1)
+        ip=$(getent hosts "$hostname" | awk '{print $1}')
+
+        while true;do
+          sleep 3
+          ip=$(getent hosts $hostname | awk '{print $1}')
+          PONG=$(redis-cli -h $(echo $hostname | cut -d'.' -f1) $AUTH_CONNECTION_STRING $TLS_CONNECTION_STRING PING)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)